src/components/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import Icon from './Icon';
+import Username from './Username';
+import ProfilePicture from './ProfilePicture';
+import ellipsis from '../assets/ellipsis.svg';
+import { classes } from '../style/comments.st.css';
+
+interface IPostOwner {
```

Comment:
Why is this also IPostOwner, i assume this is a typo

Reply:
The header needs the post owner data.

Reply:
change name!